### PR TITLE
Parent Child Layout

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

File: packages/lab/src/parent-child-layout/ParentChildItem.tsx [hunk 1]
```diff
@@ -23,10 +23,6 @@ export interface ParentChildItemProps extends HTMLAttributes<HTMLDivElement> {
    * Disable all animations.
    */
   disableAnimations?: boolean;
-  /**
-   * Direction for slide animations.
-   */
-  direction?: SlideDirection;
```
Comment (Fercas123) [inline packages/lab/src/parent-child-layout/ParentChildItem.tsx:29]: you can remove `SlideDirection` in line 15

File: packages/lab/src/parent-child-layout/ParentChildItem.tsx [hunk 2]
```diff
@@ -23,10 +23,6 @@ export interface ParentChildItemProps extends HTMLAttributes<HTMLDivElement> {
...
   /**
    * Defines the ability for an item to grow x times more compared to it's siblings, default is 0.
```
Comment (Fercas123) [inline packages/lab/src/parent-child-layout/ParentChildItem.tsx:27]:
```suggestion
   * Defines the ability for an item to grow x times more compared to its siblings, default is 0.
```

Comment (Fercas123) [reply packages/lab/src/parent-child-layout/ParentChildItem.tsx:27]: same in shrink's description

File: packages/lab/src/parent-child-layout/ParentChildItem.tsx
Comment (Fercas123) [inline packages/lab/src/parent-child-layout/ParentChildItem.tsx:1]: should we be updating these to `ComponentPropsWithoutRef`?

**Review by Fercas123 — DISMISSED** [review]

File: packages/lab/stories/parent-child-layout/parent-child-layout.stories.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+.parent-content {
+  padding: var(--salt-spacing-300);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 85px;
+  height: 100%;
+  /* border-right: var(--salt-size-border) var(--salt-separable-borderStyle) var(--salt-container-secondary-borderColor); */
```
Comment (feliciackh) [inline packages/lab/stories/parent-child-layout/parent-child-layout.stories.css:15]: nit: could remove this is not needed?

**Review by feliciackh — APPROVED** [review]

File: site/docs/components/parent-child-layout/examples.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+The parent and child component changes to a collapse view once it approaches a specified breakpoint that can be customized via the `collapsedAtBreakpoint` prop.
+
+</LivePreview>
+
+<LivePreview componentName="parent-child-layout" exampleName="ReducedMotion">
+
+### Reduced motion
+
+You can customize or disable the animations using the `prefers-reduced-motion` CSS media feature.
```
[the remark this answers is not in the document]
Comment (alycrys) [reply site/docs/components/parent-child-layout/examples.mdx:34]: no

File: site/docs/components/parent-child-layout/examples.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+</LivePreview>
+
+<LivePreview componentName="parent-child-layout" exampleName="PreferencesLayout">
+
+### Preferences Dialog
+
+This is an example of a Preferences Dialog using the parent-child layout.
```
[the remark this answers is not in the document]
Comment (alycrys) [reply site/docs/components/parent-child-layout/examples.mdx]: has been merged, so I added the link

**Review by Fercas123 — APPROVED** [review]

**Review by lilyvc — APPROVED** [review]

**Pull request merged**